networks/ropsten.json

```diff
@@ -157,7 +210,8 @@
             "underlying": "0x0A1e4D0B5c71B955c0a5993023fc48bA6E380496",
             "contract": "CErc20",
             "initial_exchange_rate_mantissa": "200000000000000000000000000",
-            "address": "0xA3C2c1618214549281E1b15dee9D682C8aa0DC1C"
+            "admin": "0x513c1Ff435ECCEdD0fDA5edD2Ad5E5461F0E8726",
```

Comment:
Do we need the admin on these? Is that generated as part of the newer deploy scripts?

Reply:
dont think we strictly need them, but i guess we add it in the newer scen code